sdk/confidentialledger/azure-security-confidentialledger/test-resources.json

```diff
@@ -41,8 +41,20 @@
 	}],
     "outputs": {
         "LEDGER_NAME": {
-            "type": "String",
+            "type": "string",
             "value": "[parameters('ledgerName')]"
-        }
+        },
+		"IDENTITYSERVICEURI": {
+			"type": "string",
+			"value": "https://identity.confidential-ledger.core.azure.com"
```

Comment:
This URI should not be hardcoded.

Reply:
Should I make it a testing constant in a separate file instead?